In [5]:
import os
import sys
import urllib.request
client_id = "uDs4RNKJANndul0wzUt6"
client_secret = "rDnVd5Xwk6"
encText = urllib.parse.quote("핀테크")
url = "https://openapi.naver.com/v1/search/blog?query=" + encText + "&display=100" # JSON 결과
# url = "https://openapi.naver.com/v1/search/blog.xml?query=" + encText # XML 결과
request = urllib.request.Request(url)
request.add_header("X-Naver-Client-Id",client_id)
request.add_header("X-Naver-Client-Secret",client_secret)
response = urllib.request.urlopen(request)
rescode = response.getcode()
if(rescode==200):
    response_body = response.read()
    print(response_body.decode('utf-8'))
else:
    print("Error Code:" + rescode)

{
	"lastBuildDate":"Mon, 31 Mar 2025 14:02:08 +0900",
	"total":456108,
	"start":1,
	"display":100,
	"items":[
		{
			"title":"<b>핀테크<\/b> 투자 후기",
			"link":"https:\/\/blog.naver.com\/kan1687\/223812724931",
			"description":"<b>핀테크<\/b> 투자, 아직도 망설이고 계신가요?  여러분 혹시 이런 고민 해보신 적 있나요? &quot;<b>핀테크<\/b> 분야가 떴다는데, 지금 시작해도 괜찮을까? 좀 더 상황을 지켜봐야 하나?&quot; 사실 이런 생각은 투자를 고민하는 우리... ",
			"bloggername":"우정잉♥",
			"bloggerlink":"blog.naver.com\/kan1687",
			"postdate":"20250329"
		},
		{
			"title":"[<b>핀테크<\/b>] 2024년 <b>핀테크<\/b> 업계 결산: 놀라운 3가지 변화",
			"link":"https:\/\/blog.naver.com\/xoom777\/223708685044",
			"description":"동안 <b>핀테크<\/b> 업계에는 예상치 못한 변화의 바람이 불었습니다. 한동안 주춤했던 트렌드가... 올 한 해 <b>핀테크<\/b> 업계를 뜨겁게 달군 3가지 트렌드에 대해 자세히 살펴보겠습니다. 암호화폐의 귀환, 기관... ",
			"bloggername":"Ich fange an",
			"bloggerlink":"blog.naver.com\/xoom777",
			"postdate":"20241229"
		},
		{
			"title":"[인사이트 캠퍼스] 2025 동계 <b>핀테크<\/b> 캠프 (with 전주대학교... ",
			"link":"https:\/\/blog.naver.com\/fininsight\/223734638126",
			"descr

In [2]:
encText

'%ED%95%80%ED%85%8C%ED%81%AC'

## URL
* http://로 시작하는 웹페이지 접속주소
* url : 실제 서비스 서버에 접속하는 주소
* parameter : 서비스에 요청하는 값
* https://search.naver.com/search.naver?ssc=tab.blog.all&query=%ED%95%80%ED%85%8C%ED%81%AC&sm=tab_opt&nso=so%3Ar%2Cp%3A1m
* url : https://search.naver.com/search.naver?
* parameter : ssc=tab.blog.all&query=%ED%95%80%ED%85%8C%ED%81%AC&sm=tab_opt&nso=so%3Ar%2Cp%3A1m